src/main/java/uk/gov/hmcts/reform/sandl/snlevents/actions/hearing/AssignSessionsToHearingAction.java

```diff
@@ -208,4 +212,17 @@ public UUID getUserTransactionId() {
     private UserTransactionData getLockedSessionTransactionData(UUID id) {
         return new UserTransactionData("session", id, null, "lock", "unlock", 0);
     }
+
+    @Override
+    public List<ActivityLog> getActivities() {
+        ActivityLog activityLog = ActivityLog.builder()
+            .userTransactionId(getUserTransactionId())
+            .id(UUID.randomUUID())
+            .entityId(relationship.getHearingId())
+            .entityName(HEARING_ENTITY)
+            .status(ActivityStatus.Listed)
+            .build();
```

Comment:
Maybe Let's create custom builder (Or at least a wrapper for Lombok's one) and mix it with fluent interface. I think the easiest to read is as follows:

```
List<ActivityLog> activityLogList = ActivityBuilder
   .userTransactionId('your-uuid')
   .withActivity('hearingId', 'Hearing', LISTED)
   .withActivity('sessionId', 'Session', ASSIGNED)
   .withActivity('sessionId', 'Session', UNASSIGNED)
   .build()
```